src/tribler-gui/tribler_gui/widgets/channelcontentswidget.py

```diff
@@ -87,6 +83,7 @@ def __exit__(self, *args):
                     obj.blockSignals(False)
 
         self.freeze_controls = freeze_controls_class
+        self.setStyleSheet("QToolTip { color: #222222; background-color: #eeeeee; border: 0px; }")
```

Comment:
Would it work when we apply this style sheet on a top-level widget (e.g., the window)? I assume that we are not going to use different styles for the `QToolTip`.

Reply:
Unfortunately, this does not work :disappointed: